Applications/CLI/Tests.cmake

```diff
@@ -371,6 +371,16 @@ if(NOT OGS_USE_MPI)
          DIFF_DATA
          h_us_quad_1000.vtu richards_pcs_0_ts_100_t_100.000000.vtu PRESSURE1 pressure
     )
+	 AddTest(
```

Comment:
tabs to be replaced with whitespaces. Please change your editor's default choice, since I have found this error already multiple times.